src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdatePreparer.java

```diff
@@ -96,6 +95,7 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
                 callbackResponse.addError("You cannot manage a fee update for this appeal");
                 break;
 
+            case PA:
```

Comment:
unsure if this change needs to be made as per the ticket - AC2 has been removed so I would expect it to still throw an error on a PA appeal if it hasn't been paid yet?